cfme/middleware/__init__.py

```diff
@@ -1,8 +1,54 @@
 from cfme.web_ui import CheckboxTable, toolbar as tb
 from functools import partial
 
+from cfme.web_ui import Table
+
 cfg_btn = partial(tb.select, 'Configuration')
 mon_btn = partial(tb.select, 'Monitoring')
 pol_btn = partial(tb.select, 'Policy')
 
 list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+class Validatable(object):
```

Comment:
@mfalesni so what do you want to do, implement it in the generic way now and then rebase this PR?